common/doc/common-doc.md

```diff
@@ -3,8 +3,15 @@
 ## Table of content
 
 * [Pool](#Pool)
+* [Grpc](#Grpc)
 
 
 ## Pool
 
 After a fork, only caller thread is activated in child process, so we mustn't join others. That's why thread container is dynamically allocated and not freed in case of fork.
+
+
+## Grpc
+The goal of the two classes provided, grpc_server_base and grpc_client_base is to create server or channel in order to use it with service. 
```

Comment:
Not clear.
I saw grpc_server_base with grpc::acceptor and grpc_client_base with grpc::connector.

And I don't understand what is the service...